src/PKSim.BatchTool/Presenters/BatchMainPresenter.cs

```diff
@@ -44,16 +53,18 @@ public void GenerateTrainingMaterial()
          start<IGenerateTrainingMaterialPresenter>();
       }
 
-      private void start<T>() where T : IInitializablePresenter<BatchStartOptions>
+      private void start<T>() where T : IBatchPresenter
       {
          var presenter = _applicationController.Start<T>();
-         _view.Hide();
+         View.Hide();
          presenter.InitializeWith(_startOptions);
       }
 
       public void GenerateProjectOverview()
       {
          start<IGenerateProjectOverviewPresenter>();
       }
+
+      public IView ActiveView => _startOptions.IsValid() ? _batchSilentView : BaseView;
```

Comment:
We need to start some kind of view in a Program.cs. I do not want to see the actual view popup so for now, I have this silent view that is minimized. Do you have a better idea?